### ISSUE TITLE
Remove PDF Earnings Export

### ISSUE THREAD

**Issue by pippinsplugins** [easydigitaldownloads/easy-digital-downloads#1808]
Currently the PDF export only supports the current year.

We need to add support for start / end year and start / end month.

**Comment by cklosowski** [comment]
@pippinsplugins @ghost1227 where do we stand with this? Now that we have a batch exporter build, are we still looking at supporting the PDFs?

**Comment by pippinsplugins** [comment]
I would still like to remove all PDF exports. They are far too large and provide very little benefit.

Replacing them with comprehensive CSV reports sounds like a much better method to me. See #3055

**Comment by pippinsplugins** [comment]
Let's remove it and release a free plugin with the feature.

**Comment by pippinsplugins** [comment]
We will need to update PDF Invoices with version checks and `file_exists` checks to PDF libraries.

**Comment by chriscct7** [comment]
As #3055 was punted to 2.6, this needs to be punted to at least 2.6

**Comment by pippinsplugins** [comment]
:+1:

**Comment by pippinsplugins** [comment]
PR submitted!

Now we need to get PDF Invoices updated. @sunnyratilal could you take care of that next week so that it conditionally loads the fpdf library if it's not available in core?

**Comment by pippinsplugins** [comment]
@ghost1227 also need to confirm that everything is fine here with PDF Stamper.

**Comment by ghost1227** [comment]
@pippinsplugins PDF Stamper no longer references any PDF functionality from core, but I'll double check...

**Comment by chriscct7** [comment]
FES also need stop be checked but I think we've been bundling our own
library as well

**Comment by sunnyratilal** [comment]
PDF Invoices uses tcpdf which is already included in the plugin. Upgraded from fpdf to TCPDF back when 2.0 was released so removing this from Core won't affect PDF Invoices at all.

**Comment by sunnyratilal** [comment]
@pippinsplugins `googlechartlib` also can be removed from `includes/libraries` as it was only used for the PDF reports and isn't used anywhere else in Core.

**Comment by pippinsplugins** [comment]
@sunnyratilal nice, thanks.

@chriscct7 FES includes its own copy of fPDF but it does load googlechartlib from core:

```php
/**
 * Make PDF chart.
 *
 * Creates chart used in FES pdf.
 *
 * @since 2.3.0
 * @access public
 *
 * @param FES_Vendor $vendor Vendor to make PDF for.
 * @return GoogleChart Chart to put on pdf.
 */
function fes_draw_chart_image( $vendor ) {
	require_once EDD_PLUGIN_DIR . '/includes/libraries/googlechartlib/GoogleChart.php';
	require_once EDD_PLUGIN_DIR . '/includes/libraries/googlechartlib/markers/GoogleChartShapeMarker.php';
	require_once EDD_PLUGIN_DIR . '/includes/libraries/googlechartlib/markers/GoogleChartTextMarker.php';
.....
```
Let's move those to FES and get an update pushed out asap. @chriscct7

**Comment by pippinsplugins** [comment]
I'm going to go ahead and merge this into `release/2.7` so that we can more easily identify issues in the coming months.

**Comment by pippinsplugins** [comment]
PR submitted to remove them from FES too.

**Comment by cklosowski** [comment]
Ok from what i'm seeing, we're good on all the dependencies @pippinsplugins @ghost1227 @sunnyratilal @chriscct7 correct? If so then I think we can make sure this is closed out properly, once and for all with 2.7.

**Comment by ghost1227** [comment]
No problems on my end!

**Comment by pippinsplugins** [comment]
Yep. FES has been updated already.

**Comment by sunnyratilal** [comment]
Yep all ready for merge in 2.7

**Comment by pippinsplugins** [comment]
Merged into `release/2.7` already.
